<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>David Ehrenberg</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>trtoolbox</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>SVD</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>102</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pb_singular">
        <property name="text">
         <string>Singular values</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>SVD Components</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_svd_svd">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="value">
         <number>5</number>
        </property>
        <property name="displayIntegerBase">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_dosvd">
        <property name="text">
         <string>SVD</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>82</width>
       <height>143</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="cb_svd_contour">
        <property name="text">
         <string>Contour</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_svd_3d">
        <property name="text">
         <string>3D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_svd_spectra">
        <property name="text">
         <string>Spectra</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_svd_traces">
        <property name="text">
         <string>Traces</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_svd_plot">
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>31</width>
       <height>161</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <zorder>line</zorder>
    <zorder>verticalLayoutWidget_2</zorder>
    <zorder>verticalLayoutWidget_3</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Raw data</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>82</width>
       <height>143</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cb_data_contour">
        <property name="text">
         <string>Contour</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_data_3d">
        <property name="text">
         <string>3D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_data_spectra">
        <property name="text">
         <string>Spectra</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_data_traces">
        <property name="text">
         <string>Traces</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_data_plot">
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Truncation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Frequency</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_trunc_freq">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_trunc_time">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_truncate">
        <property name="text">
         <string>Truncate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>31</width>
       <height>161</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <zorder>line_3</zorder>
    <zorder>verticalLayoutWidget</zorder>
    <zorder>verticalLayoutWidget_8</zorder>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>121</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>SVD Components</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_gf_svd">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="value">
        <number>5</number>
       </property>
       <property name="displayIntegerBase">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Time constants</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="txt_gf_tcs">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1e-6, 1e-5, 1e-4, 1e-3, 1e-2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_dogf">
       <property name="text">
        <string>Global Fit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Result</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="txt_gf_results"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>521</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Global analysis</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>85</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QCheckBox" name="cb_gf_contour">
        <property name="text">
         <string>Contour</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_gf_3d">
        <property name="text">
         <string>3D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_gf_spectra">
        <property name="text">
         <string>Spectra</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_gf_traces">
        <property name="text">
         <string>Traces</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_gf_profile">
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_gf_das">
        <property name="text">
         <string>EAS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_gf_plot">
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>41</width>
       <height>201</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <zorder>line_2</zorder>
    <zorder>verticalLayoutWidget_6</zorder>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>160</y>
      <width>118</width>
      <height>313</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QCheckBox" name="cb_lda_results">
       <property name="text">
        <string>Results</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_lda_lcruve">
       <property name="text">
        <string>L-curve</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Alpha value:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_lda_plot_alpha">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_lda_contour">
       <property name="text">
        <string>Contour</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_lda_3d">
       <property name="text">
        <string>3D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_lda_spectra">
       <property name="text">
        <string>Spectra</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_lda_traces">
       <property name="text">
        <string>Traces</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_lda_map">
       <property name="text">
        <string>LDA Map</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_lda_solution">
       <property name="text">
        <string>Solution vector</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_lda_plot">
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>331</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>LDA</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Number:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Number:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_lda_tc_number">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_lda_alpha_bottom">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>0.1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Upper limit:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_lda_alpha_upper">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Lower limit:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Lower limit:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_lda_dolda">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>LDA</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_lda_alpha_number">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Time constants</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Alpha values</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_lda_tc_upper">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>1e-1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_lda_tc_bottom">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>1e-7</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Upper limit:</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>31</width>
       <height>311</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_lda_use_svd">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Use SVD data</string>
     </property>
    </widget>
    <zorder>line_4</zorder>
    <zorder>label_9</zorder>
    <zorder>label_8</zorder>
    <zorder>txt_lda_tc_number</zorder>
    <zorder>txt_lda_alpha_bottom</zorder>
    <zorder>label_12</zorder>
    <zorder>txt_lda_alpha_upper</zorder>
    <zorder>label_10</zorder>
    <zorder>label_6</zorder>
    <zorder>pb_lda_dolda</zorder>
    <zorder>txt_lda_alpha_number</zorder>
    <zorder>label_5</zorder>
    <zorder>label_11</zorder>
    <zorder>txt_lda_tc_upper</zorder>
    <zorder>txt_lda_tc_bottom</zorder>
    <zorder>label_7</zorder>
    <zorder>cb_lda_use_svd</zorder>
   </widget>
   <widget class="QPushButton" name="pb_plt_close">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Close all Plots</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_file_data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Data file:
*.dat</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_file_time">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Time file:
*.dat</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_file_freq">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Freq file:
*.dat</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_save_results">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>540</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save results</string>
    </property>
   </widget>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>verticalLayoutWidget_4</zorder>
   <zorder>verticalLayoutWidget_5</zorder>
   <zorder>verticalLayoutWidget_7</zorder>
   <zorder>pb_plt_close</zorder>
   <zorder>label_file_data</zorder>
   <zorder>label_file_time</zorder>
   <zorder>label_file_freq</zorder>
   <zorder>pb_save_results</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Load Data</string>
    </property>
    <addaction name="actionLoad_all"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_time"/>
    <addaction name="actionLoad_frequency"/>
    <addaction name="actionLoad_data"/>
    <addaction name="actionSet_delimiter"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_test_data"/>
   </widget>
   <widget class="QMenu" name="menuChange_Units">
    <property name="title">
     <string>Change Units</string>
    </property>
    <addaction name="actionTime_change"/>
    <addaction name="actionFreq_change"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionClear_all"/>
   </widget>
   <addaction name="menuData"/>
   <addaction name="menuChange_Units"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_all">
   <property name="text">
    <string>Load all</string>
   </property>
  </action>
  <action name="actionLoad_time">
   <property name="text">
    <string>Load time</string>
   </property>
  </action>
  <action name="actionLoad_frequency">
   <property name="text">
    <string>Load frequency</string>
   </property>
  </action>
  <action name="actionLoad_data">
   <property name="text">
    <string>Load data</string>
   </property>
  </action>
  <action name="actionLoad_test_data">
   <property name="text">
    <string>Generate data</string>
   </property>
  </action>
  <action name="actionTime_change">
   <property name="text">
    <string>Time</string>
   </property>
  </action>
  <action name="actionFreq_change">
   <property name="text">
    <string>Frequency</string>
   </property>
  </action>
  <action name="actionFrequency_long">
   <property name="text">
    <string>Frequency long</string>
   </property>
  </action>
  <action name="actionFrequency_short">
   <property name="text">
    <string>Frequency short</string>
   </property>
  </action>
  <action name="actionSet_delimiter">
   <property name="text">
    <string>Set delimiter</string>
   </property>
  </action>
  <action name="actionClear_all">
   <property name="text">
    <string>Clear all</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
